carbonmark-api/helpers/fetchProjectDetails/index.js

```diff
@@ -0,0 +1,21 @@
+import { fetchGSProject } from "./fetchGSProject";
+import { fetchVCSProject } from "./fetchVCSProject";
+
+export const fetchProjectDetails = async ({ key }) => {
+  try {
+    const [registry, registryProjectId] = key.split("-");
```

Comment:
**nit:** Perhaps the string manipulation should be outside of this function so it's not tied to the format of project keys?